src/Pages/ProductList/ProductItem/ProductItem.scss

```
@@ -1,4 +1,4 @@
-.itemContainer {
+.productItem {
   width: 20%;
   padding-bottom: 40px;
   padding: 0 0 40px 20px;
```

```suggestion
  padding-bottom: 40px;
  padding: 0 0 40px 20px;
```
padding-bottom은 padding에 의해 무시될 것 같습니다. 필요 없으면 지워주세요.